### ISSUE TITLE
Closing / killing chromedriver.exe when using Selenium::Chrome

### ISSUE THREAD

**Issue by Qarj** [gempesaw/Selenium-Remote-Driver#251]
Apologies if this has been covered somewhere, but I couldn't find it.

When I issue the following command, a new Chrome window and chromedriver.exe open fine.
```
                $driver = Selenium::Chrome->new (binary => 'C:\selenium-server\chromedriver.exe',
                                             'extra_capabilities' => {'chromeOptions' => {'args' => ['window-size=1260,968']}}
                                             );
```

I am able to then run my various selenium commands fine.

When I am done, I issue the following command:
```
my $selresp = $driver->quit(); 
```

The Chrome browser window shuts down fine, but chromedriver.exe remains open. How do you shut down chromedriver.exe?

**Comment by gempesaw** [comment]
hmm, looking at `Binary.pm`, it looks like there's a method `shutdown_binary` that gets called when the `$driver` var goes out of scope. But, if the `$driver` goes out of scope during global destruction, it won't get called, since all bets are off during global destruction.

You could try either of these:

- calling `$driver->shutdown_binary`
- forcing $driver out of scope, probably setting it to `undef` would work

If those work, it probably makes sense to make `shutdown_binary` get called automatically after `quit` if the `$driver` was started in binary mode.

Thanks!

**Comment by Qarj** [comment]
Yes it works!

calling `$driver->shutdown_binary();` after quit did the trick.

I even tried the following bit of code to ensure all windows were closed before calling quit, but it didn't help and is not needed when using shutdown_binary:

```
    my $close_handles = $sel->get_window_handles;
    
    for my $close_handle (reverse 0..@{$close_handles}) {
       print {*STDOUT} "Shutting down window $close_handle\n";
       $driver->switch_to_window($close_handles->[$close_handle]);
       $driver->close();
    }
```

Thanks very much for such a quick response, I wasn't expecting it. This will be very useful to us since we run many testing sessions in parallel. This means a system wide purge of chromedriver.exe is not possible, so I'm very happy that there is a solution to this.

**Comment by gempesaw** [comment]
Thanks again for bringing this up, I've added notes to the documentation that suggest using `shutdown_binary`. Cheers!
